crate/cli/src/actions/certificates/certify.rs

```diff
@@ -65,7 +65,7 @@ pub struct CertifyAction {
     #[clap(long = "days", short = 'd', default_value = "365")]
     number_of_days: usize,
 
-    /// The path to a X509 extension's file, containing a `v3_ca` parag
+    /// The path to a X509 extension's file, containing a `v3_ca` param
```

Comment:
I really meant `parag`, a short for paragraph

Reply:
eventually, could we change it to `v3_ca section`?

Reply:
reverted